lib/MultiCompiler.js

```diff
@@ -208,12 +233,12 @@ module.exports = class MultiCompiler {
 							if (err) handler(err);
 							if (stats) {
 								allStats[compilerIdx] = stats;
-								compilerStatus[compilerIdx] = "new";
-								if (compilerStatus.every(Boolean)) {
+								compilerStatus[compilerIdx] = STATUS_NEW;
```

Comment:
This change are optimizations for V8 or just a more consistent approach?

Reply:
Maybe you can add a typedef CompilerStatus = STATUS_PENDING | STATUS_NEW | STATUS_DONE and type `compilerStatus` to make it more clear.